local-volume/helm/provisioner/templates/provisioner.yaml

```diff
@@ -0,0 +1,89 @@
+{{- $configMapName := .Values.configmap.configMapName }}
+{{- $provisionerNamespace := .Values.common.namespace }}
+{{- $provisionerName := .Values.daemonset.name }}
+{{- $imageFull := .Values.daemonset.image }}
+{{- $imagePullPolicy := .Values.daemonset.imagePullPolicy }}
+{{- $defaultMountDir := .Values.common.defaultMountDir }}
+{{- $serviceAccount := .Values.daemonset.serviceAccount }}
+{{- $kubeConfigEnabled := .Values.daemonset.kubeConfigEnabled }}
+{{- $kubeConfigEnv := .Values.daemonset.kubeConfigEnv }}
+{{- $nodeLabelsEnabled := .Values.daemonset.nodeLabelsEnabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configMapName }} 
+  namespace: {{ $provisionerNamespace }} 
+data:
+{{- if $nodeLabelsEnabled }}
+  nodeLabelsForPV: |
+    - failure-domain.beta.kubernetes.io/zone
+    - failure-domain.beta.kubernetes.io/region
+{{- end }}
+  storageClassMap: |
+#
+# Populating entries with user provided disk discovery directories 
+# and mount directories per storage class.
+#
+{{- range $tmp, $val := .Values.configmap.storageClass }}
+   {{- range $key, $class := $val }}
```

Comment:
$key => $storageClass to be more clear.  And maybe $class => $classConfig

Reply:
done